.gitignore

```diff
@@ -3,3 +3,5 @@ node_modules
 dist
 package-lock.json
 coverage
+./package-lock.json
+.env
```

Comment:
Is there anything you added to .env? If yes, you need to add `.env.example` file that contains an explanation of what's supposed to be added e.g `DB_USERNAME=your database username`

Reply:
@Ruqoyah  solved.